libs/design-system-components/src/utils/iconByAppType.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
iconByAppType is actually a component so no need to put it in the utils. Maybe create a generic component like ExtensionIcon and also add a story

Reply:
Aha, yyea. Will do that now

Reply:
another question: why is this not in the ds-core?

Reply:
Most of the components requiring it were in DS components. That's why I kept it there

Reply:
I think it will be best to move it to ds-core - adjust typings

Reply:
alright, that works